### PR TITLE
Deprecate DefaultPageLayout and PageWrapper for _app and PageLayout

### LINKED ISSUE #3790: Works V2 page sending 2 pageviews in GA
Not 100% sure why, need to make sure we fix this before launching.

### PR DESCRIPTION
**Pull request by jamesgorrie**
Combo of the learning from #3791 and #3789 

Fixes #3790 

We're going to be removing `DefaultPageLayout` and `PageWrapper` in favour of `PageLayout` and `_app`. 

`_app` will be the relatively consistent pieces of the site, and `PageWrapper` will be the parts of the interface and metadata that need to respond to each piece of content.

It's relatively similar to what we have, which is good, but more consistent, tidy, and inline with normal practice with the new next.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev I'm going to call this phase 1.

As I haven't touched `DefaultPageLayout` or `PageWrapper` - this will still work for the `content` app.

It's already a pretty massive PR, so I might leave it there?

File: catalogue/webapp/package.json [hunk 1]
```diff
@@ -31,6 +32,7 @@
     "raw-loader": "^0.5.1",
     "react": "^16.7.0-alpha.2",
     "react-dom": "^16.7.0-alpha.2",
+    "react-ga": "^2.5.3",
     "react-markdown": "^4.0.3",
     "react-test-renderer": "^16.4.1",
     "react-transition-group": "^2.4.0",
```
Comment (jamesgorrie) [inline catalogue/webapp/package.json]: I'll be able to remove these deps when `_app` makes it into `common`.

File: catalogue/webapp/pages/works.js [hunk 2]
```diff
@@ -41,7 +38,15 @@ export const Works = ({
   handleSubmit,
   version
 }: ComponentProps) => (
-  <Fragment>
+  <PageLayout
+    title={`${query ? `${query} | ` : ''}Catalogue search`}
```
Comment (davidpmccormick) [inline catalogue/webapp/pages/works.js]: @hthair the page titles on `/works` are going to change with this PR. What used to be

`Image catalogue search${query ? `: ${query}` : ''}`

will become

`${query ? `${query} | ` : ''}Catalogue search`

e.g.

`Image catalogue search: eggs` will become `eggs | Catalogue search`

**Comment by jamesgorrie** [comment]
@gestchild sorry, I was going to wait for both reviews - I'll wait for your review before going any further with this though - trigger happy is all.

**Comment by gestchild** [comment]
@jamesgorrie it's fine by me too

**Pull request merged**